vignettes/ORFikOverview.Rmd

```diff
@@ -87,4 +87,100 @@ There are 7 exons in this group, named 3_* where star is the orf number.
 You will also see that the second orf named 3_2 is twice,
 because that single orf is split on two exons.
 
+# Cage-data for tissue specific 5' utrs
 
+In the previous example we used the reference annotation of the 5' utrs
+from the package GenomicFeatures.
+This is a general reference, which will fail to catch tissue specific orfs.
+Here is an example how to include cage-data, to set new Transcription start sites(TSS).
+
+```r
+library(ORFik)
+library(GenomicFeatures)
+
+# Make BSGenome  example with fasta file and gtf file
+
+# First the gtf file, we will extract the 5' leaders and reassign them
+
+# here we use example data from GenomicFeatures,
+# usually you have your own txdb object, a gtf file etc.
+
+samplefile <- system.file("extdata", "hg19_knownGene_sample.sqlite", package = "GenomicFeatures")
+txdb <- loadDb(samplefile)
+fiveUTRs = fiveUTRsByTranscript(txdb, use.names = TRUE) # <- extract only 5' leaders
+cageData <- "./cageData/heartCage.bed" # location of the cage file
```

Comment:
You can not make examples of code that is not runnable. Make example cage small bed and put it into inst/extdata, data for tests put into inst/testdata. And make sure to load data in examples same way as above with system.faile

Reply:
This package will crash, because vignettes are build-ed during install.